src/backend/aspen/workflows/import_gisaid/save.py

```diff
@@ -78,5 +89,61 @@ def cli(
         print(f"Successfully imported {num_rows}")
 
 
+def write_table(metadata_fh, pathogen_slug: str, public_repository_name: str):
+    data = csv.DictReader(metadata_fh, delimiter="\t")
+
+    interface: SqlAlchemyInterface = init_db(get_db_uri(Config()))
+    fields_to_import = {
+        "strain": "strain",
+        "pango_lineage": "lineage",
+        "gisaid_epi_isl": "isl",
+        "region": "region",
+        "country": "country",
+        "division": "division",
+        "location": "location",
+    }
+    num_rows = 0
+    with session_scope(interface) as session:
+        pathogen = session.query(Pathogen).filter(Pathogen.slug == pathogen_slug).one()  # type: ignore
+        repository = session.query(PublicRepository).filter(PublicRepository.name == public_repository_name).one()  # type: ignore
+
+        dest_table = PublicRepositoryMetadata.__table__
+        temp_table = create_temp_table(session, dest_table)
+
+        objects: List[
+            Dict[str, Union[Optional[str], Optional[int], Optional[datetime.datetime]]]
+        ] = []
+        # We insert into a temporary table and then swap table contents with public_repository_metadata
```

Comment:
why?

Reply:
This is the simplest way to make sure all of our old metadata gets replaced with new metadata - we write all the new data to a temp table, then delete all the SC2 rows from the main metadata table, then copy all the data from the temp table into our main metadata table inside a transaction. There's not a lot of (potentially buggy) code to write to compare all of this data row-by-row.